packages/react-native/src/primitives/RadioGroup/styles.ts

```diff
@@ -1,12 +1,15 @@
 import { StyleSheet } from 'react-native';
 
+import { StrictTheme } from '../../theme';
 import { RadioGroupStyles } from './types';
 
-export const styles: RadioGroupStyles = StyleSheet.create({
-  label: {
-    fontSize: 18,
-    fontWeight: '500',
-    marginHorizontal: 4,
-    marginVertical: 2,
-  },
-});
+export const getThemedStyles = (theme: StrictTheme): RadioGroupStyles => {
+  const { components } = theme.tokens;
+
+  return StyleSheet.create({
```

Comment:
We could add a `container` property to `RadioGroupStyle` for custom theme `container` styles, what do you think?

Reply:
I like that idea